src/security-rules/security-rules-api-client.ts

```diff
@@ -119,6 +124,38 @@ export class SecurityRulesApiClient {
       });
   }
 
+  public listRulesets(pageSize: number = 100, pageToken?: string): Promise<ListRulesetsResponse> {
```

Comment:
do you even want to allow users to customize this number?

Reply:
I see your point. This was approved during the API review process on the grounds that other similar APIs in the admin SDK (e.g. `auth.listUsers()`) allow setting custom page sizes. So we are being consistent here. But I agree. Most users won't need it.